ShadowmapTechnologies/LocationBrowserPlugin
# enable safari extension

To test the extension:

- Run `Shadowmap Location.xcodeproj`
- Make sure you select your target (Mac, iOS), it support all platforms
<img width="239" alt="Screen Shot 2022-03-02 at 12 58 56 PM" src="https://user-images.githubusercontent.com/6974081/156339264-9234234d-43ad-4e6e-9da5-7a3661bf83b2.png">
- Click Run
- You would say something like that
<img width="425" alt="Screen Shot 2022-03-02 at 1 02 41 PM" src="https://user-images.githubusercontent.com/6974081/156339982-d71124a6-e4f6-416b-9595-8c9dc5f781ef.png">

- Click open Safari Extention and enable it by doing the checkmark.

<img width="796" alt="Screen Shot 2022-03-02 at 1 03 58 PM" src="https://user-images.githubusercontent.com/6974081/156340207-5dcafe11-1212-4cac-ac83-a21b56e14cff.png">

- Now for Debugging you can from the Developer Menu.
<img width="656" alt="Screen Shot 2022-03-02 at 1 05 00 PM" src="https://user-images.githubusercontent.com/6974081/156340364-5a823d5e-8110-483a-9e74-9e4a36d7ff3d.png">
